server/remote/msgraph/get_default_calendar_view.go

```diff
@@ -51,8 +51,8 @@ func (c *client) GetDefaultCalendarView(remoteUserID string, start, end time.Tim
 
 func (c *client) DoBatchViewCalendarRequests(allParams []*remote.ViewCalendarParams) ([]*remote.ViewCalendarResponse, error) {
 	requests := []*singleRequest{}
-	if !c.checkUserStatus() {
-		c.Logger.Warnf(LogUserInactive)
+	if c.tokenHelpers != nil && !c.tokenHelpers.CheckUserStatus(c.Logger, c.mattermostUserID) {
```

Comment:
`c.tokenHelpers` and `c.mattermostUserID` will never be defined here. This is the only function that they won't be defined